page_objects/examples/weather_shopper_mobile_app/weather_shopper_payment_page.py

```diff
@@ -0,0 +1,77 @@
+"""
+Page object for the payment page in Weathershopper application.
+"""
+# pylint: disable = W0212,E0401
+import conf.locators_conf as locators
+from utils.Wrapit import Wrapit
+from core_helpers.mobile_app_helper import Mobile_App_Helper
+
+class WeatherShopperPaymentPage(Mobile_App_Helper):
+    "Page objects for payment page in Weathershopper application."
+
+    @Wrapit._exceptionHandler
+    @Wrapit._screenshot
+    def enter_valid_payment_details(self, payment_details):
+        "This method is to enter valid payment details"
+
+        result_flag = self.click_element(locators.payment_method_dropdown)
```

Comment:
Use methods created in this page object.

Reply:
Using the new methods